src/backend/tests/models/test_charge.py

```diff
@@ -1,12 +1,27 @@
 import unittest
 
-from datetime import date
+from datetime import date, datetime, timedelta
 from dateutil.relativedelta import relativedelta
 from expungeservice.models.disposition import Disposition
 from tests.factories.charge_factory import ChargeFactory
 from tests.factories.case_factory import CaseFactory
 
 
+class ChargeTypeTestsParent(unittest.TestCase):
+    def setUp(self):
+        self.charge_dict = ChargeFactory.default_dict()
+        self.charges = []
+        last_week = datetime.today() - timedelta(days=7)
+        self.convicted = Disposition(ruling="Convicted", date=last_week)
+        self.dismissed = Disposition(ruling="Dismissed", date=last_week)
+        self.diverted = Disposition(ruling="Diverted", date=last_week)
+        self.unrecognized_disposition = Disposition(ruling="Something unrecognized", date=last_week)
+        self.no_complaint = Disposition(ruling="No Complaint", date=last_week)
```

Comment:
We should just extract these as constants like

`CONVICTED = Disposition(ruling="Convicted", date=LAST_WEEK)`

and then just call them with `SomeClass.CONVICTED`